src/app/Reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
```

Comment:
combine multiple reducers together
only have one here but we will can have multiple reducers to manage more parts of the application